userdocs/diagnostics/protocol-type-non-conformance.md

```diff
@@ -0,0 +1,41 @@
+# Protocol type not conforming to itself
+Swift disallows us from using a protocol as a type that conforms to itself as illustrated in the examples below
```

Comment:
Educational notes are written in a didactic style that start from most general to specific.

I'd suggest taking a look at the related note about [protocols with associated types](https://github.com/apple/swift/blob/373dcc76ec9968212f27c5baab49e924bcd62770/userdocs/diagnostics/associated-type-requirements.md), which starts off with an introduction that's very relevant here:

> Protocols in Swift may be used as types

Perhaps something like the following text would be helpful:

> Protocols in Swift may be used as types, but a protocol type (sometimes called an 'existential type') does not conform to its own protocol.

Because this particular question has been asked many times, you can find several different explanations on the Swift Forums. It might be helpful to see which ones seem to have the greatest clarity, and to reach out to those authors to see if they'd be willing to help contribute their work to this note.

Reply:
Thanks @xwu  for the feedback, I will work on them.